### ISSUE TITLE
Adapt storefinder to work properly with new a11y e2e tests

### ISSUE THREAD

**Issue by Matejk00** [SAP/cloud-commerce-spartacus-storefront#6676]
Right now new a11y tests from #6281 are failing. We have to adapt storefinder code to make them green.

**Comment by Matejk00** [comment]
QA steps:
- Run all storefinder a11y e2e tests from `cypress\integration\accessibility\tabbing-order.e2e-spec.ts` and check if all of them are passing.

**Comment by mirkosoft** [comment]
Just one test didn't pass but as it's probably because of sampledata change let's agree this QA is passed and handle sampledata issue in the below separate ticket: https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/6998
